src/pages/billing/Project.js

```diff
@@ -86,7 +87,13 @@ const WorkspaceCard = Utils.memoWithName('WorkspaceCard', ({ workspace, isExpand
             'aria-controls': isExpanded ? id : undefined,
             'aria-owns': isExpanded ? id : undefined,
             style: { display: 'flex', alignItems: 'center' },
-            onClick: onExpand
+            onClick: () => {
+              Ajax().Metrics.captureEvent(Events.billingProjectExpandWorkspace, {
```

Comment:
is this an async call?

Reply:
It is, but we don't care about the response.